credentials/alts/internal/conn/record_test.go

```diff
@@ -41,10 +41,10 @@ func Test(t *testing.T) {
 }
 
 var (
-	nextProtocols   = []string{"ALTSRP_GCM_AES128"}
+	recordProtocols = []string{"ALTSRP_GCM_AES128_REKEY"}
```

Comment:
Make a const for `"ALTSRP_GCM_AES128_REKEY"` and use it in both `recordProtocols` and `altsRecordFuncs`

Also, is this string used in other places? If so, change them all to use this const they are guaranteed to be the same.

Reply:
Created const.

The string is used in another package `credentials/alts/internal/handshaker/handshaker.go`. We can create a const for both but that's a bigger change than the scope of this PR :) Also, this string is not expected to change because it's the same in ALTS-C++ and ALTS-Java.